test/JDBC/input/forjson/forjsonauto-vu-prepare.sql

```diff
@@ -139,7 +139,7 @@ END
 GO
 
 CREATE FUNCTION forjson_vu_f_1()
-RETURNS sys.NVARCHAR(5000) AS
+RETURNS sys.NVARCHAR(4000) AS
```

Comment:
If a user has an already created function with typmod value as 5000 it would still be executed as before.
But if the user tries to create a new function now, it won't be able to create such a function with a typmod value >4000 for nvarchar. This change handles the limit of typmod for various datatypes such as 'sys.varchar', 'sys.nvarchar', 'sys.varbinary', 'sys.nchar','sys.binary','sys.bpchar'.